src/applications/vaos/components/AppointmentCard/index.unit.spec.jsx

```diff
@@ -46,7 +46,10 @@ describe('VAOS Component: AppointmentCard', () => {
     const wrapper = renderWithStoreAndRouter(
       <AppointmentCard appointment={appointment}>
         <h1 className="vads-u-margin-y--2p5">
-          <AppointmentDateTime appointment={appointment} />
+          <AppointmentDateTime
```

Comment:
nit: hmm this is suspicious since it looks like the only usage of AppointmentDateTime is in this unit test. Maybe we should just remove this component completely?

Reply:
Agreed.